common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -869,7 +869,7 @@ def test_submit_problem_error_with_codejail_exception(self):
             # Ensure that the user is NOT staff
             module.system.user_is_staff = False
 
-            # Simulate a codejail exception 'Exception: test error'
+            # Simulate a codejail exception "Exception: test's errored"
```

Comment:
Nit: My punctuation alarms are ringing. Shouldn't it be `tests errored` without the apostrophe?

Reply:
I'll try to come up with a grammatically correct test. =)